pkg/noobaaaccount/reconciler.go

```diff
@@ -373,6 +388,33 @@ func (r *Reconciler) CreateNooBaaAccount() error {
 		r.Logger.Errorf("got error on NooBaaAccount creation. error: %v", err)
 		return err
 	}
+	// create join secret conatining auth token for remote noobaa account
+	annotationValue, exists := util.GetAnnotationValue(r.NooBaa.GetAnnotations(), "remote-operator")
```

Comment:
If the `remote-operator` annotation exist, do you need to also create access keys (`r.Secret`)?

Maybe just reuse the same `r.Secret` and populate it with the relevant values according to the annotatioin?

Reply:
it's a valid point but since it was Kaustav's decision to use a new secret, i'm not sure what led to him creating a new secret.
Lets leave the new secret in place and when Kaustav returns from his leave we can change this. i'll open an issue for it as well (since changing it will most likely cause changes to code in other repos such as ocs-operator which uses this secret)

Reply:
I prefer to make the code simpler and not to create 2 secrets for a single noobaa account. I suggest that for now add an `"auth_token"` key on `r.Secret`. If there is a reason to create a separate secret than we can make this change later